utils/env.ts

```diff
@@ -0,0 +1,12 @@
+import * as path from "path";
+
+export const isWin = process.platform.startsWith("win");
+
+// in *nix `rm` by default skip non-existing files,
+// on win* we have to use another command and check if directory exists
+export const rmCommand = (pathToRemove: string) =>
+  isWin
+    ? `if exist "${path}" rmdir /s /q "${pathToRemove
```

Comment:
How does it works? I've read a little bit about `path` but I find completely different examples. 😞 
Also maybe `getRemoveCommand` is better name for this function?

Reply:
done